### ISSUE TITLE
SyntaxError: Non-ASCII character '\xc2' in file ./tbpurge.py on line 4

### ISSUE THREAD

**Issue by bentolor** [sebastianst/tbpurge#1]
Thanks for publishing this tool, @sebastianst  This is exactly what I'm currently looking for and I was about to hack one on my own and luckily found your one.

I'm currently trying to get it running. Unfortunately It throws me an error on running it either directly on my QNap NAS and a Ubuntu 19.04 installation:

````
$ ./tbpurge.py
  File "./tbpurge.py", line 4
SyntaxError: Non-ASCII character '\xc2' in file ./tbpurge.py on line 4, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
````

**Comment by sebastianst** [comment]
Ah I see, line 4 has the copyright character in it  (©). Line 1 sets the interpreter to `/usr/bin/env python`. What's the default python on your system, that is, what is `python --version` for you?
It may make sense to change the first line to `/usr/bin/env python3` to explicitly select python version 3, which this script requires. Please try that out and let me know if this fixes your issue.

**Comment by bentolor** [comment]
Thanks for your blazingly fast reply. I just tried to fix it with 
````
# -*- coding: utf-8 -*-
````

That lead me to run into a different problem:

````
./tbpurge.py
  File "./tbpurge.py", line 46
    assert isdir(args.path), f"{args.path} is not a directory"
                                                             ^
SyntaxError: invalid syntax
 ▲  ben@neptun /m/t/h/b/A/TitaniumBackup.2019-06  $ ./tbpurge.py -h
  File "./tbpurge.py", line 46
    assert isdir(args.path), f"{args.path} is not a directory"
                                                             ^
SyntaxError: invalid syntax
````

`/usr/bin/env python --version` leads to `Python 2.7.16` which comes a little to suprise to me: I expected python3 would be the default in newer Ubuntu.

Changing to `python3` fixes the issue… Oh this unfortunate Python2/3 pity…

**Comment by sebastianst** [comment]
OK, I will switch it to `python3` then. I always forget that on most systems, `python` is usually not version 3. I use Arch Linux, where it is.

**Comment by sebastianst** [comment]
closed by 422e2ffabfaa382f5bc28c01bef0f3f431172d6b
